Config/Config.hs

```diff
@@ -14,4 +19,20 @@ config = do
     option customTailwind
 
     -- Static directory.
-    initStaticDirStorage
\ No newline at end of file
+    initStaticDirStorage
+
+    -- Private and public keys to sign and verify image style URLs.
+    privateKeyContent <- liftIO $ readRsaKeyFromFile "./Config/id_rsa"
+    publicKeyContent <- liftIO $ readRsaKeyFromFile "./Config/id_rsa.pub"
+
+    case (readRsaSecret privateKeyContent, readRsaPublicKey publicKeyContent) of
+        (Just privateKey, Just publicKey) -> option $ RsaPublicAndPrivateKeys publicKey privateKey
```

Comment:
@mpscholten I've created the files with `ssh-keygen -t rsa -f ./Config/id_rsa` which creates both the private and public files.

But here I'm getting `Nothing`.

According to their example of using `ssh-keygen -t rsa` I should get [this header](https://github.com/puffnfresh/haskell-jwt/blob/4390e2e13ab16ae6d7421c8c49a399e0db80aa22/src/Web/JWT.hs#LL405C18-L405C39) (BEGIN **RSA** PRIVATE KEY), but mine is 

-----BEGIN **OPENSSH** PRIVATE KEY-----

How did you create your public and private keys in your code?

Reply:
I also have `-----BEGIN RSA PRIVATE KEY-----` in my private key.

I've used the code from this gist https://gist.github.com/ygotthilf/baa58da5c3dd1f69fae9